docs/configuration/configuration.md

```diff
@@ -718,6 +718,9 @@ tls_config:
 # The host to use if the container is in host networking mode.
 [ host_networking_host: <string> | default = "localhost" ]
 
+# Whether filter networks by user defined network mode if the container has multiple networks.
+[ filter_networks: <boolean> | default = true ]
```

Comment:
(During Prometheus bug scrub meeting): This flag seems too fuzzy e.g. "filter" can both mean filter out, and filter in (choose). Also this option is doing more than that (if no user defined network then something else is performed).